spec/controllers/attachments_controller_spec.rb

```diff
@@ -46,7 +46,7 @@
 
   before { allow(User).to receive(:current).and_return user }
 
-  describe :destroy do
+  describe '#destroy' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.